salesagility/SuiteCRM
# Add Anniversary option to date field AOW

## Description



Adds the option to choose Anniversary as date comparison in AOW.
The condition will run if the current month and day are equal to the field's value.
Currently all operators and value comparisons have no effect.

![image](https://user-images.githubusercontent.com/34056696/44779722-26317200-ab78-11e8-9735-24f1b4b8e883.png)


## Motivation and Context

Allows easy sending of birthday emails and other workflow related to anniversaries.

## How To Test This


1. Create a workflow.
2. Make a condition of type Equal To, Date, Anniversary
3. See the workflow fires both on save and from the scheduler.

## Types of changes


### Final checklist
